airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```diff
@@ -88,6 +94,10 @@ definitions:
         type: string
       password:
         type: string
+        default: ""
```

Comment:
Why would we have an empty string by default here? Based on a quick check, my guess is that `InterpolatedString` does not take `None` as an input. If this is the case, does `None` and an empty string means the same thing for Basic Auth?

Reply:
The general reasoning was that the [basic http authenticator implementation](https://github.com/airbytehq/airbyte/blob/master/airbyte-cdk/python/airbyte_cdk/sources/streams/http/requests_native_auth/token.py#L68) (even prior to low-code) was a concatenated string of username + password that was base64 encoded. So rather than have to translate password None into "" in that code, we just opted for the empty string default. And that carried into our low code implementation